docs/governance/governance.mdx

```diff
@@ -14,174 +14,54 @@ import useBaseUrl from '@docusaurus/useBaseUrl'
 
 ## What is Helium Governance?
 
-The Helium Network is the world's largest continuous decentralized wireless network. The process by
-which the Network can be modified, whether in its technical architecture, the tokens' economic
-model, or meta-governance changes, is governed by the global community of Helium token owners.
-
-Helium token (HNT, IOT, MOBILE) owners include Hotspot owners, Network users, wireless network
-operators, hardware manufacturers, and more.
-
-Tokens are created by providing some form of work for the Network. For example, transmitting data
-packets through the internet on behalf of a network user. Hotspot owners provide public, accessible
-wireless coverage. Network users must acquire and burn tokens to send data over the network. Tokens
-must be locked for voting on network changes, ensuring long-term stakeholder alignment.
-
-Anyone participating in the Helium ecosystem can submit a Helium Improvement Proposal (HIP) as a
-first step to modifying the Network. All HIPs are published in the public Helium GitHub repository,
-then discussed and debated in public forums, including the Helium community Discord server, and on
-monthly Community Calls. Any Helium token owner is eligible to vote on HIPs. To vote, a Helium token
-owner must first lock their tokens for a period of time, up to four years, which grants the owner
-voting power determined by the number of locked tokens and the lock duration. A 2/3 supermajority of
-the quorum passes proposals.
+Helium governance exists to manage the process by which the network can be modified, whether in its
+technical architecture, the tokens' economic model, or meta-governance changes. It is governed by
+the global community of Helium token owners. These token owners include Hotspot deployers, network
```

Comment:
instead of "Helium token owners", "owners of Helium tokens (HNT, MOBILE, IOT)" would be better, as people else might misunderstand "Helium token owners" as "owners of the Helium token". Also clarifies, which tokens are relevant for governance (DCs are not). Maybe its only unclear for non-native English speakers like me.